src/main/java/io/jenkins/plugins/checks/github/GitHubChecksContext.java

```diff
@@ -75,7 +76,13 @@
      * @return the commit sha of the run or null
```

Comment:
If `null` is allowed (which I think is a bad choice) then we must mark the method with `Nullable`